WalletWasabi/Hwi/Models/HardwareWalletModels.cs

```diff
@@ -9,19 +9,65 @@ namespace WalletWasabi.Hwi.Models
 	/// </summary>
 	public enum HardwareWalletModels
 	{
+		[Description("Hardware Wallet")]
 		Unknown,
+
+		[Description("Coldcard")]
 		Coldcard,
+
+		[Description("Coldcard Simulator")]
 		Coldcard_Simulator,
+
+		[Description("BitBox")]
 		DigitalBitBox_01,
+
+		[Description("BitBox Simulator")]
 		DigitalBitBox_01_Simulator,
+
+		[Description("KeepKey")]
 		KeepKey,
+
+		[Description("KeepKeySimulator")]
 		KeepKey_Simulator,
+
+		[Description("Ledger Nano S")]
 		Ledger_Nano_S,
+
+		[Description("Trezor One")]
 		Trezor_1,
+
+		[Description("Trezor One Simulator")]
 		Trezor_1_Simulator,
+
+		[Description("Trezor T")]
 		Trezor_T,
+
+		[Description("Trezor T Simulator")]
 		Trezor_T_Simulator,
+
+		[Description("BitBox")]
 		BitBox02_BTCOnly,
+
+		[Description("BitBox")]
 		BitBox02_Multi,
 	}
+
+	public static class EnumExtensions
+	{
+		public static T? GetFirstAttribute<T>(this Enum value) where T : Attribute
+		{
+			var type = value.GetType();
+			var memberInfo = type.GetMember(value.ToString());
+			var attributes = memberInfo[0].GetCustomAttributes(typeof(T), false);
+
+			return attributes.Any() ? (T)attributes[0] : null;
```

Comment:
This is not covered by a test I think. It probably should because at this point I'm not sure what happens when that Enum does not actually have that attribute. Presumably it throws some exception on line 61. If it is by design, then it's better to document it by simply saying:

```csharp
/// <summary><param name="value" /> must contain attribute <typeparamref name="T"/> or an exception is thrown.</summary>
```

Reply:
If the enum doesn't have the Description attribute, the `memberInfo[0].GetCustomAttributes(typeof(T), false)` line will return with an empty list.

Reply:
Right. I have missed that.